File: Codopy/lab7 PM/lab01.py
#01 Is x an Integer?

#input
i = float(input())
#process
integer =  i - int(i) 
if integer == 0 :
    i = f'{i:.0f}'
    n = ''
else:    
    i = f'{i}'
    n = 'not '
#output
print(f'{i} is {n}an integer.')

File: CodoC++/Lab03 While For Loop/Lab01.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    //input
    char yen[10];
    fgets(yen,10,stdin);

    int yenchar = atoi(yen);
    //process
    while ( yenchar >= 0){
        //output
        printf("%d \n",yenchar);
        yenchar = yenchar - 1;
    }
}   